Dockerfile

```diff
@@ -57,3 +57,6 @@ ADD /scripts/ /scripts/
 
 ADD tripal_chado_install /scripts/setup.d/50tripal_chado_install
 ADD tripal_apache.conf /etc/apache2/conf-enabled/tripal_apache.conf
+
+ENV TRIPAL_CORE_MODULES="tripal_views tripal_db tripal_cv tripal_organism tripal_analysis tripal_feature"
```

Comment:
Hum, yes I was hesitating to do it like this, I see no use case where people would not want them